Kubernetes/flannel/overlay/README.md

```diff
@@ -4,8 +4,9 @@
 * Build Kube Proxy from [PR 70896](https://github.com/kubernetes/kubernetes/pull/70896) and copy to c:\k
 * Copy Kubeconfig from Linux master to c:\k
 * Download the following files to c:\k
-    [start.ps1](https://github.com/Microsoft/SDN/raw/master/Kubernetes/flannel/overlay/start.ps1) 
-    [helper.psm1](https://github.com/Microsoft/SDN/raw/master/Kubernetes/windows/helper.psm1) 
+  * [start.ps1](https://github.com/Microsoft/SDN/raw/master/Kubernetes/flannel/overlay/start.ps1) 
```

Comment:
you probably mean ``https://github.com/Microsoft/SDN/raw/master/Kubernetes/flannel/start.ps1``, because the file you mention will be deleted in this PR.

Reply:
Yes, these need updating indeed. I was going to update the readme.md's after we finalized the decision where we want to keep the files (or revert the previous commit alternatively)